src/main/java/net/sf/jabref/importer/fileformat/MedlinePlainImporter.java

```diff
@@ -1,289 +1,279 @@
+                    if ((parts.length > 1) && !parts[1].isEmpty()) {
+                        hm.put("month", parts[1]);
+                    }
+                } else if (lab.equals("MH") || lab.equals("OT")) {
+                    if (!hm.containsKey("keywords")) {
+                        hm.put("keywords", val);
+                    } else {
+                        String kw = hm.get("keywords");
+                        hm.put("keywords", kw + ", " + val);
+                    }
+                } else if (lab.equals("CON") || lab.equals("CIN") || lab.equals("EIN") || lab.equals("EFR")
+                        || lab.equals("CRI") || lab.equals("CRF") || lab.equals("PRIN") || lab.equals("PROF")
+                        || lab.equals("RPI") || lab.equals("RPF") || lab.equals("RIN") || lab.equals("ROF")
+                        || lab.equals("UIN") || lab.equals("UOF") || lab.equals("SPIN") || lab.equals("ORI")) {
+                    if (!comment.isEmpty()) {
+                        comment = comment + "\n";
+                    }
+                    comment = comment + val;
+                }
+                //                // Added ID import 2005.12.01, Morten Alver:
+                //                else if (lab.equals("ID"))
+                //                    hm.put("refid", val);
+                //                    // Added doi import (sciencedirect.com) 2011.01.10, Alexander Hug <alexander@alexanderhug.info>
+                else if (lab.equals("AID")) {
+                    String doi = val;
+                    if (doi.startsWith("doi:")) {
+                        doi = doi.replaceAll("(?i)doi:", "").trim();
+                        hm.put("doi", doi);
+                    }
+                }
+            }
+            // fix authors
+            if (!author.isEmpty()) {
+                author = AuthorList.fixAuthor_lastNameFirst(author);
+                hm.put("author", author);
+            }
+            if (!editor.isEmpty()) {
+                editor = AuthorList.fixAuthor_lastNameFirst(editor);
+                hm.put("editor", editor);
+            }
+            if (!comment.isEmpty()) {
+                hm.put("comment", comment);
+            }
+
+            BibtexEntry b = new BibtexEntry(DEFAULT_BIBTEXENTRY_ID, EntryTypes
+                    .getBibtexEntryType(type)); // id assumes an existing database so don't
+
+            // Remove empty fields:
+            ArrayList<Object> toRemove = new ArrayList<>();
+            for (String key : hm.keySet()) {
+                String content = hm.get(key);
+                if ((content == null) || content.trim().isEmpty()) {
+                    toRemove.add(key);
+                }
+            }
+            for (Object aToRemove : toRemove) {
+                hm.remove(aToRemove);
+            }
+
+            // create one here
+            b.setField(hm);
+            bibitems.add(b);
+
+        }
+
+        return bibitems;
+
+    }

```

Comment:
Please check the codecov reports, as you do not have 100% code coverage in this importer. 

And also check why the git diff tries to replace all lines in this file. Probably wrong setting with line endings.